server/utils/recipeUtils.js

```diff
@@ -15,13 +15,57 @@ const getByName = asyncHandler(async (recipeName, userID, Recipe) => {
     return recipe
 })
 
-const getByID = asyncHandler(async (recipeID, Recipe) => {
+const getByID = asyncHandler(async (recipeID, userID, Recipe) => {
     await Recipe.sync()
-    const recipe = await Recipe.findOne({ where: { id: recipeID } })
+    const recipe = await Recipe.findOne({ where: { id: recipeID, userID: userID } })
```

Comment:
recipeID is already unique so userID is not needed here

Reply:
Resolved.